includes/admin/class-amp-editor-blocks.php

```diff
@@ -177,6 +195,14 @@ public function enqueue_block_editor_assets() {
 		}
 	}
 
+	/**
+	 * Enqueues the front-end block stylesheet.
+	 */
+	public function enqueue_block_assets() {
+		$stylesheet_base = 'amp-blocks';
+		wp_enqueue_style( $stylesheet_base . '-style', amp_get_asset_url( "/css/{$stylesheet_base}.css" ), array(), AMP__VERSION );
```

Comment:
We should not enqueue this unless it is actually needed. So what that means is that we should call `wp_enqueue_style()` _inside_ the `render_callback`. As an added benefit, browsers now will load the stylesheet asynchronously.

Reply:
Great idea, https://github.com/ampproject/amp-wp/pull/1916/commits/06a519c2b7b4e8d87d73ddd3fd44fdaaa35cc6b4 enqueues the stylesheet in the `render_callback`.